### PR TITLE
Skip inactive org accounts; include account-name filter

### PR DESCRIPTION
**Pull request by lyricnz**
Partial solution for https://github.com/Innablr/revolver/issues/246

### CODE REVIEW

File: revolver-config-example.yaml [hunk 1]
```diff
@@ -140,6 +140,8 @@ defaults:
 # List of organizational accounts, used to discover the full account list
 organizations:
   - accountId: "000000000000"
+    # optional regular expression to filter organization accounts
+    accountNameRegex: "-nonprod$"
```
Comment (alutman-innablr) [inline revolver-config-example.yaml:144]: Also add to README.md.

File: lib/config-schema.ts [hunk 2]
```diff
@@ -184,6 +184,7 @@ const ConfigSchema = z
         z
           .object({
             accountId: AWSAccountId,
+            accountNameRegex: z.string().optional(),
```
Comment (alutman-innablr) [inline lib/config-schema.ts:187]: Minor idea, could also add regex to the exclude logic by either specifying accountId or accountRegex in excludeList.

**Review by alutman-innablr — CHANGES_REQUESTED** [review]
One idea, not required.

README update requested.

File: revolver-config-example.yaml (hunk 1 above)
Comment (lyricnz) [reply revolver-config-example.yaml:144]: Added

File: lib/config-schema.ts (hunk 2 above)
Comment (lyricnz) [reply lib/config-schema.ts:187]: Could do. No need to add it until required.

Comment (lyricnz) [reply lib/config-schema.ts:187]: Could do. No need to add it until required.

**Pull request merged**